components/camel-spring-boot/src/main/java/org/apache/camel/spring/boot/actuate/endpoint/CamelRoutesEndpoint.java

```diff
@@ -76,6 +74,11 @@ public void startRoute(String id) throws Exception {
         camelContext.getRouteController().startRoute(id);
     }
 
+    public void resetRoute(String id) throws Exception {
+        ManagedRouteMBean resetRoute = camelContext.getManagedRoute(id, ManagedRouteMBean.class);
+        resetRoute.reset();
```

Comment:
Hi Claus, 

With some changes, we can see get users to see, 
2017-07-28 14:28:00.391  INFO 6507 --- [  XNIO-3 task-1] org.apache.camel.impl.RouteService       : {"headers":{},"body":"The provided route id does not exists","statusCode":"NO_CONTENT","statusCodeValue":204}
2017-07-28 14:28:05.983  INFO 6507 --- [1 - timer://bar] bar                                      : From timer (bar) ...
2017-07-28 14:28:05.983  INFO 6507 --- [2 - timer://foo] foo                                      : From timer (foo) ...

when curl -XPOST -s http://localhost:8080/camel/routes/info/reset which has the incorrect 'id'. 

Since the start/stop/resume/ do not explicitly display anything,  we can show a message like above for 'reset' if the users provided an incorrect id. 

public String resetRoute(String id) throws Exception {
        ManagedRouteMBean resetRoute = camelContext.getManagedRoute(id, ManagedRouteMBean.class);
        if (resetRoute != null) {
            resetRoute.reset(true);
        } else {
            String description = "The provided route id does not exists";
            ResponseEntity<?> objectResponseEntity = ResponseEntity.status(HttpStatus.NO_CONTENT).body(description);
            ObjectMapper objectMapper = new ObjectMapper();
            return objectMapper.writeValueAsString(objectResponseEntity);
        }
        return null;
    }

Let me know if this is fine or acceptable.